### PR TITLE
add: Input Component

### PR DESCRIPTION
**Pull request by kavyakunder**
Input Component consists of :

1.] Simple Input 

2.] Varients of Input:
  a.) Simple
  b.)Outline
  c.)Flushed
  d.)Unstyled

3.]Radio Buttons
4.]Checkboxes

https://user-images.githubusercontent.com/81958043/152498470-fff962a8-f622-46ad-8e22-1eb2346c5c91.mp4

### CODE REVIEW

File: Components/Input/input.css [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+  margin: 1rem;
+  padding: 1.5rem;
+  border-radius: 10px;
+}
+label {
+  font-size: 1rem;
+  padding: 1rem;
+}
+.basic {
+  font-size: 1rem;
```
Comment (stutisk) [inline Components/Input/input.css]: you can make a seperate variable for font size ,as its common in line 13 and 17

**Review by stutisk — COMMENTED** [review]
else code looks good to me

**Review by shuklahimanshu81 — COMMENTED** [review]
Ready to be merged. Looks good to me.

File: Components/Input/input.css (hunk 1 above)
Comment (kavyakunder) [reply Components/Input/input.css]: Done!

**Pull request merged**